.circleci/config.yml

```diff
@@ -177,7 +177,7 @@ workflows:
 #          env: "preprod"
 #          context:
 #            - hmpps-common-vars
-#            - court-case-test-app-preprod
+#            - hmpps-template-typescript-preprod
 #          requires:
 #            - request-preprod-approval
 #      - request-prod-approval:
```

Comment:
You can probably remove this prod jobs entirely as we've got no intention of deploying there.

The preprod one is fine commented out for now, I assume your plan is to uncomment once dev is working?

Reply:
yep :)